CHANGES.md

```diff
@@ -1,6 +1,13 @@
 # Change Log
 
-## v0.8.13
```

Comment:
In #56, [`pack.yaml` was bumped to `v0.9.0`](https://github.com/StackStorm-Exchange/stackstorm-vsphere/pull/56/files#diff-860549dd65a84fe07d27ed9c80a96ccaR6), but the [`CHANGES.md` entry was for `v0.8.13`](https://github.com/StackStorm-Exchange/stackstorm-vsphere/pull/56/files#diff-8b1c3fd0d4a6765c16dfd18509182f9dR3).

This PR now fixes that situation.

@armab Since you care a lot about changelog entries, please advise. The two questions I have for you are:

1. Are you okay with rewriting previous changelog entries?
2. Do you want the changelog entry fix split out into its own PR?

Reply:
I think it's fine to either way for this pack repo if we'll have eventually actual/synced changelog.